less/bookmarksToolbar.less

```diff
@@ -12,7 +12,7 @@
   display: flex;
   flex: 1;
   padding: 3px 10px;
-  height: 23px;
+  height: 30px;
```

Comment:
Conditionally making it 30px would be easy :smiley: For reference, Chrome is 27px (between the url bar and the bottom of bookmarks toolbar). In Brave though, we have a bordered style above the bookmarks toolbar, which makes it harder to fix this in reasonably (padding, etc).

There are also some borders:
* class style "navigatorWrapper" has 1px border-bottom w/ #aaa
* class style "bookmarksToolbar" has 1px border-top w/ #fff

In addition to conditionally setting the size to 30px, I'd like to propose conditionally eliminating these two borders *IF* the bookmarks tb is shown (it's needed when not using the bookmarks tb, since there is no other border below)